.bazelrc

```diff
@@ -45,31 +36,21 @@ build:remote --define=EXECUTOR=remote
 build:remote --remote_executor=grpcs://remotebuildexecution.googleapis.com
 build:remote --remote_timeout=3600
 
-# Minimize what is downloaded
-build:inmemory --experimental_inmemory_jdeps_files
-build:inmemory --experimental_inmemory_dotd_files
-
-# Minimize what is downloaded
-build:toplevel --config=inmemory
-build:toplevel --experimental_remote_download_outputs=toplevel
-
-build:minimal --config=inmemory
-build:minimal --experimental_remote_download_outputs=minimal
-
 # --google_credentials=some_file.json
 build:remote --google_default_credentials=true
 build:remote --config=toplevel
 
 run:remote --experimental_remote_download_outputs=all --noexperimental_inmemory_jdeps_files --noexperimental_inmemory_dotd_files
 
-# Compose the remote configs with an instance name
-# A couple examples below:
+# configs to minimize what is downloaded
+build:inmemory --experimental_inmemory_jdeps_files
+build:inmemory --experimental_inmemory_dotd_files
 
-# --config=ci-instance adds the instance name
-build:ci-instance --remote_instance_name=projects/k8s-prow-builds/instances/default_instance
+build:toplevel --config=inmemory
+build:toplevel --experimental_remote_download_outputs=toplevel
 
-# Config we want to use in ci
-build:ci --config=remote --config=ci-instance
```

Comment:
Can we please keep a `build:ci` option, even if it does nothing? That way we can (continue) to define `bazel test --config=ci` jobs, and just change what this means over time.

`build:ci --noshow_progress` is a good value to add for non-interactive jobs